scripts/create_assignments_from_csv.py

```diff
@@ -115,17 +118,25 @@ def main(arguments):
     for worker in workers:
         workers_dict[worker.user_id] = worker
 
+    if not (args.x_field and args.y_field):
+        addresses = batch_geocode(locations, out_sr=args.wkid)
     assignments_to_add = []
+    i = 0
```

Comment:
Consider using enumerate to get the index and object without needing to explicitly define `i`

Reply:
imo `enumerate` is less accessible to people that doesn't use Python frequently and I still have to manually iterate here because the loop being used in assignments_to_csv

Reply:
I think it looks un-pythonic. What do you mean, can't you just do

`for i, assignment in enumerate(assignments_in_csv):`?

Reply:
Ahhhh lol okay i thought you wanted to enumerate the addresses. Yes the indexes are the same, will fix